packages/server/src/api/routes/tests/datasource.spec.ts

```diff
@@ -285,9 +286,6 @@ describe("/datasources", () => {
           [FieldType.STRING]: {
             name: stringName,
             type: FieldType.STRING,
-            constraints: {
-              presence: true,
-            },
```

Comment:
Ended up removing this because when we create the table, we don't set DB-level not-null checks. I'm uncertain why, but adding this here and then having an exclusion for it later on in the `objectContaining` check seemed weird.